config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -37,99 +38,157 @@ presubmits:
       timeout: 5h
     path_alias: sigs.k8s.io/cloud-provider-azure
     branches:
-      - master
+    - master
     labels:
       preset-dind-enabled: "true"
       preset-kind-volume-mounts: "true"
       preset-azure-cred-only: "true"
       preset-azure-anonymous-pull: "true"
       preset-azure-capz-sa-cred: "true"
     extra_refs:
-      - org: kubernetes-sigs
-        repo: cluster-api-provider-azure
-        base_ref: release-1.15
-        path_alias: sigs.k8s.io/cluster-api-provider-azure
-        workdir: true
+    - org: kubernetes-sigs
+      repo: cluster-api-provider-azure
+      base_ref: release-1.15
```

Comment:
Using the community cluster requires CAPZ's release-1.16 or main branch. Updating that here should get the tests further along.